editor.planx.uk/src/@planx/components/Pay/Editor.tsx

```diff
@@ -284,10 +287,11 @@ const Component: React.FC<Props> = (props: Props) => {
                 </Link>{" "}
                 for more details.
               </Typography>
+              <Typography variant="subtitle2" sx={{ mb: 2 }}>Any values beginning with @ will be dynamically read from the applicant's passport.</Typography>
```

Comment:
nit: maybe "Any values beginning with @ will be dynamically read from data values set throughout the flow" ? I'm not sure all editors are familiar with "passport" language (Editor modals often use "Data value" placeholder) and "applicant's passport" may not clearly indicate that "invisible" nodes like SetValue can be referenced here too?

Reply:
This works well for me!

Reply:
Copy updated here: [aff4d67](https://github.com/theopensystemslab/planx-new/pull/3015/commits/aff4d670a169ac917c882cd6c6a6aeaaeb96afce)